src/styles/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin shadow() {
+  filter: drop-shadow(1px 1px $theme-shadow-color);
+}
```

Comment:
folder needs to be rechecked and mixin needs to be added to https://github.com/rayanfer32/nexus-market-cap/pull/19 respective folder